plugins/wazo-gigaset/N510/templates/base.tpl

```diff
@@ -113,6 +113,8 @@
      Redirect device always to own provisioning Server so you are in control for example: Firmware updates 	-->
     {%- if http_port %}
     <SYMB_ITEM ID="BS_IP_Data1.aucS_DATA_SERVER[0]" class="symb_item" value='"{{ ip }}:{{ http_port }}"'/>
+    <SYMB_ITEM ID="BS_IP_Data1.aucS_USER_FIRMWARE_URL" class="string" value='"{{ ip }}:{{ http_port }}/42/fw262.bin"' />
+    <S_CONFIGFILE_URL class="string" value='"{{ ip }}:{{ http_port }}/{{ XX_mac_addr }}.xml"'/>
```

Comment:
Why do you enforce `S_CONFIGFILE_URL` ?
For now it works without, so I wonder :)
Many thx 👍